Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -321,9 +321,12 @@ - (void)performSwitchRequest:(NSURL *)appSwitchURL paymentType:(BTPayPalPaymentT
 
     NSURLComponents *urlComponents = [NSURLComponents componentsWithURL:appSwitchURL resolvingAgainstBaseURL:NO];
 
+    // Verify what bt_int_type does
     NSString *queryForAuthSession = [urlComponents.query stringByAppendingString:@"&bt_int_type=2"];
     urlComponents.query = queryForAuthSession;
 
+    // Verify this URL matches v4 URLs
```

Comment:
✅ I compared the URLs constructed for v4 vs v5, and they are structured the same with the same query params.